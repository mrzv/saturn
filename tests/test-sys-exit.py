import sys

#---#

print("Hello")

#---#

sys.exit(0)

#---#

print("Past the end")
