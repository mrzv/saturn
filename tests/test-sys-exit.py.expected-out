>>> import sys                                                                  


>>> print("Hello")                                                              

Hello

>>> sys.exit(0)                                                                 

────────────────────────────── Caught SystemExit ───────────────────────────────
