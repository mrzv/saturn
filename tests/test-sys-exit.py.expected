import sys

#---#

print("Hello")

#o> Hello
#---#

sys.exit(0)

#---#

print("Past the end")
